CMake/xrt.cmake

```diff
@@ -3,19 +3,29 @@
 
 # Bring in XRT git submodule and exclude parts of XRT we don't need
 set(XRT_EXCLUDE_SUB_DIRECTORY
+  src/runtime_src/core/pcie/emulation
+  src/runtime_src/core/pcie/windows
   src/runtime_src/core/pcie/driver
-# Hard dependencies for xrt cmake target are introduced in master. Can't exclude below directories any more
-# src/runtime_src/core/pcie/emulation
-# src/runtime_src/xocl
   src/runtime_src/core/pcie/tools
-  src/runtime_src/core/pcie/windows
+  src/runtime_src/core/pcie/noop
+
   src/runtime_src/core/tools
+  src/runtime_src/core/edge
+
+  src/runtime_src/tools
+  src/runtime_src/xocl
   src/runtime_src/ert
   src/runtime_src/xrt
-  src/xma
+
   tests/validate
+  src/xma
```

Comment:
xma is no longer is XRT, you can remove this line

Reply:
Good to know. I'll remove it.